helm/cluster/templates/clusterapi/controlplane/_helpers_flatcar.tpl

```diff
@@ -1,86 +1,39 @@
-{{- define "cluster.kubeadmControlPlane.kubeadmConfigSpec.ignition.containerLinuxConfig.additionalConfig.systemd.units" }}
-{{- range $.Values.internal.controlPlane.kubeadmConfig.ignition.containerLinuxConfig.additionalConfig.systemd.units }}
-- name: {{ .name }}
-  {{- if hasKey . "enabled" }}
-  enabled: {{ .enabled }}
-  {{- end }}
-  {{- if hasKey . "mask" }}
-  mask: {{ .mask }}
-  {{- end }}
-  {{- if .contents }}
-  contents: |
-    {{- .contents | trim | nindent 4 }}
-  {{- end }}
-  {{- if .dropins }}
-  dropins:
-  {{- range .dropins }}
-  - name: {{ .name }}
-    {{- if .contents }}
-    contents: |
-      {{- .contents | trim | nindent 6 }}
-    {{- end }}
-  {{- end }}
-  {{- end }}
+{{- define "cluster.internal.controlPlane.kubeadm.ignition" }}
```

Comment:
Renamed template to align naming with other ones, and changed implementation to just pass the values to common Flatcar helpers that render systemd and storage config for Ignition.